charts/promtail/templates/prometheus-rules.yaml

```
@@ -0,0 +1,23 @@
+{{- if and .Values.serviceMonitor.enabled .Values.serviceMonitor.prometheusRule.enabled -}}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: {{ template "promtail.fullname" . }}
+{{- if .Values.serviceMonitor.prometheusRule.namespace }}
+  namespace: {{ .Values.serviceMonitor.prometheusRule.namespace | quote }}
+{{- end }}
```

Fix indention and use `with`:

```suggestion
  {{- with .Values.serviceMonitor.prometheusRule.namespace }}
  namespace: {{ . | quote }}
  {{- end }}
```